### ISSUE TITLE
Reload environment getting called multiple times

### ISSUE THREAD

**Issue by namaaror** [mockoon/mockoon#651]
**Describe the bug**
Mockoon reload environment gets triggered multiple times (with success toast) [Environment "Demo API" was modified externally and reload] for every change that is made in the UI or in the environment file.

**To Reproduce**
Steps to reproduce the behavior.
Open the mockoon application via custom protocol in the browser. eg. mockoon://load-environment?url=<env_path>, the dialog will appear to save the environment. Now when we make the change in the saved environment file or the UI (eg. to the description) the mockoon app starts to trigger multiple success toast messages.

**Expected behavior**
The reload should happen only once.

**Screenshots**

**Mockoon version:**
1.17.0

**OS / OS version:**
Mac OS

**Comment by 255kb** [comment]
This appears to be a duplicate of #649
